Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN DEBIAN_FRONTEND=noninteractive \
     make -j${CPUS} world && make install-world && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-# Remove wget
+# Remove wgetcd 
```

Comment:
I think it's a typo but better to revert it so that in future while taking a look on history does not create some kind of confusion.